pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -702,6 +800,59 @@ func takeByTopologyNUMAPacked(topo *topology.CPUTopology, availableCPUs cpuset.C
 	return cpuset.New(), fmt.Errorf("failed to allocate cpus")
 }
 
+// takeByTopologyUnCoreCachePacked uses the "packed" sorting strategy similar to takeByTopologyNUMAPacked.
+// It includes an additional level of sorting by uncorecache
+func takeByTopologyUnCoreCachePacked(topo *topology.CPUTopology, availableCPUs cpuset.CPUSet, numCPUs int, cpuSortingStrategy CPUSortingStrategy) (cpuset.CPUSet, error) {
```

Comment:
we should merge this logic in `takeByTopologyPacked` rather than duplicate the function

Reply:
this module is not part of the kubernetes golang exported packages, usablie by third parties. And this function is private anyway :)